i18n/fr.toml

```diff
@@ -42,10 +42,10 @@ other = "Mise à jour"
 # website header SEO
 ##############################
 ["meta[title]"]
-other = "Dogecoin - An open-source peer-to-peer digital currency"
+other = "Dogecoin - Une monnaie numérique pair-à-pair open-source"
```

Comment:
Peer-to-peer? Ou on laisse la traduction pair-a-pair

Reply:
Je propose qu'on garde pair-à-pair et qu'on standardise une bonne fois pour toute pour éviter de le changer à nouveau tous les ans.